morse_code.rb

```diff
@@ -0,0 +1,64 @@
+class MorseCodeDecoder
+  MORSE_CODE = {
+    '.-' => 'A',
+    '-...' => 'B',
+    '-.-.' => 'C',
+    '-..' => 'D',
+    '.' => 'E',
+    '..-.' => 'F',
+    '--.' => 'G',
+    '....' => 'H',
+    '..' => 'I',
+    '.---' => 'J',
+    '-.-' => 'K',
+    '.-..' => 'L',
+    '--' => 'M',
+    '-.' => 'N',
+    '---' => 'O',
+    '.--.' => 'P',
+    '--.-' => 'Q',
+    '.-.' => 'R',
+    '...' => 'S',
+    '-' => 'T',
+    '..-' => 'U',
+    '...-' => 'V',
+    '.--' => 'W',
+    '-..-' => 'X',
+    '-.--' => 'Y',
+    '--..' => 'Z'
+  }.freeze
+
+  def self.decode_char(code)
+    MORSE_CODE[code]
+  end
+
+  def self.decode_word(word)
+    decoded_word = ''
+    characters = word.split
+    characters.each { |char| decoded_word << decode_char(char) }
+    decoded_word
+  end
+
+  def self.decode(message)
+    decoded_message = ''
+    words = message.split
+    words.each { |word| decoded_message << decode_word(word) << ' ' }
+    decoded_message.strip
+  end
```

Comment:
- [ ] Your result is different than the correct result, when it decodes the message it has a space between each character instead of having space between words. Please kindly fix that 🙏🏾 

Your result

<img width="540" alt="Screenshot 2023-05-11 at 22 53 43" src="https://github.com/shafiuyushawu/morse-code-decode/assets/74501523/db09850a-dda2-40be-a0e3-aae0547ae92c">

The correct result

```
"A"
"MY"
"MY NAME"
"A BOX FULL OF RUBIES"
```